packages/apollo-react-relay-duct-tape-compiler/package.json

```diff
@@ -41,7 +41,7 @@
   },
   "peerDependencies": {
     "graphql": "^15.0.0",
-    "@graphitation/supermassive-ast": "^2.0.1",
+    "@graphitation/supermassive": "^3.0.1",
```

Comment:
@freiksenet @vladar This change makes it hard to ingest newer duct-tape-compiler builds into TMP, as we still install supermassive v2 in the root node_modules atm.